ratis-netty/src/main/java/org/apache/ratis/netty/client/DataStreamReplyDecoder.java

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
I'm not sure about this,  most project use `/**`

Reply:
See https://www.apache.org/licenses/LICENSE-2.0
>  the text should be enclosed in the appropriate comment syntax for the file format. 

It requires comment syntax.  If we use /**, the license will be shown in the generated javadoc.   It probably is not something that we want.

Reply:
+1 for `/*`.  With `/**` IntelliJ IDEA also warns about "Dangling Javadoc comment".